src/app/pages/debrief/components/vehicle-checks-card/vehicle-checks-card.html

```diff
@@ -9,7 +9,7 @@ <h4 class="fault-heading">{{ 'debrief.vehicleChecks' | translate }}</h4>
       <ion-row *ngFor="let question of tellMeShowMeQuestions" class="mes-data ion-align-items-center">
           <ion-col size="32" class="ion-no-padding"></ion-col>
           <ion-col size="2" class="ion-align-self-start ion-no-padding">
-            <div class="counter-icon">
+              <div class="boolean-icon">
```

Comment:
Can we name this class something more specific e.g. ```vehicle-check-icon```

And then update the .scss file please